Jenkinsfile_CNP

```diff
@@ -6,7 +6,81 @@ def type = "java"
 def product = "ccd"
 def component = "message-publisher"
 
+// Variables to switch pipeline logic and wiring per type of build
+def definitionStoreDevelopPr = "PR-575" // This doesn't change frequently, but when it does, only change this value.
+
+// vars needed for functional tests
+// Assume a feature build branched off 'develop', with dependencies develop-to-develop.
+env.DEFINITION_STORE_HOST = "http://ccd-definition-store-api-${definitionStoreDevelopPr}.service.core-compute-preview.internal".toLowerCase()
```

Comment:
Think we need to add the logic to set `DEFINITION_STORE_HOST` to AAT when appropriate?

Reply:
done